### PR TITLE
#3: CI build

### LINKED ISSUE #3: CI build
## Acceptance Criteria

- [ ] CI Gradle build added
- [ ] CI Link checking added

### PR DESCRIPTION
**Pull request by redcatbear**
Closes #3.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,14 +31,14 @@ Editor support for writing and maintaining OpenFastTrace documents
 
 ### Information for Users
 
-* [User Guide](doc/user_guide/user_guide.md)
+<!-- * [User Guide](doc/user_guide/user_guide.md) -->
```
Comment (ckunki) [inline README.md:34]:
```suggestion
```

File: README.md [hunk 2]
```diff
@@ -31,14 +31,14 @@ Editor support for writing and maintaining OpenFastTrace documents
...
 * [Changelog](doc/changes/changelog.md)
 
 ### Information for Contributors
 
 Requirement, design documents and coverage tags are written in [OpenFastTrace](https://github.com/itsallcode/openfasttrace) format.
 
-* [Developer Guide](doc/developer_guide/developer_guide.md)
+<!-- * [Developer Guide](doc/developer_guide/developer_guide.md) -->
```
Comment (ckunki) [inline README.md:41]:
```suggestion
```

File: src/test/java/org/itallcode/openfasttrace/intelijplugin/MockTest.java [hunk 3]
```diff
@@ -0,0 +1,10 @@
+package org.itallcode.openfasttrace.intelijplugin;
+
+import org.junit.Test;
+
+public class MockTest {
+    @Test
+    public void testNothing(){
```
Comment (ckunki) [inline src/test/java/org/itallcode/openfasttrace/intelijplugin/MockTest.java:7]:
```suggestion
    public void testNothing() {
```

Comment (ckunki) [reply src/test/java/org/itallcode/openfasttrace/intelijplugin/MockTest.java:7]: Could you maybe add a comment explaining the purpose of the test?

File: README.md [hunk 4]
```diff
@@ -6,17 +6,17 @@
 
 [![Build Status](https://github.com/itsallcode/openfasttrrace-intellij-pluginactions/workflows/ci-build.yml/badge.svg)](https://github.com/itsallcode/openfasttrrace-intellij-pluginactions/workflows/ci-build.yml)
 
-[![Quality Gate Status](https://sonarcloud.io/api/project_badges/measure?project=com.exasol%3Aexasol-virtual-schema-lua&metric=alert_status)](https://sonarcloud.io/dashboard?id=com.exasol%3Aexasol-virtual-schema-lua)
```
Comment (ckunki) [inline README.md:9]: Would it make sense or is it technically possible to use project-keeper, here?

**Review by ckunki — CHANGES_REQUESTED** [review]

File: README.md (hunk 4 above)
Comment (redcatbear) [reply README.md:9]: It's a gradle-based project unfortunately. PK does not support Gradle.

**Review by kaklakariada — APPROVED** [review]

File: src/test/java/org/itallcode/openfasttrace/intelijplugin/MockTest.java [hunk 5]
```diff
@@ -3,6 +3,8 @@
 import org.junit.Test;
 
 public class MockTest {
+    // This is a test that only exists to have a single test result in the test report. It is required to find out if
+    // the CI build works. It will be removed with a later PR once real tests exist.
     @Test
```
Comment (ckunki) [inline src/test/java/org/itallcode/openfasttrace/intelijplugin/MockTest.java:8]: 👍

**Review by ckunki — APPROVED** [review]

**Pull request merged**